interactions/api/models/message.py

```diff
@@ -100,7 +100,11 @@ def __init__(self, **kwargs):
             if self._json.get("timestamp")
             else datetime.utcnow()
         )
-        self.author = User(**self._json.get("author"))
+        self.author = (
+            User(**self._json.get("author"))
+            if self._json.get("author")
+            else User(**self._json.get("user"))
```

Comment:
```suggestion
            else None
```

I realized that us trying to get the `user` field will not work here either. Cool Discord, cool.